src/common/function/src/scalars/json/json_path_exists.rs

```diff
@@ -159,20 +146,20 @@ mod tests {
                          } if valid_types ==
             vec![
                 TypeSignature::Exact(vec![
-                    ConcreteDataType::json_datatype(),
-                    ConcreteDataType::string_datatype(),
+                    DataType::Binary,
```

Comment:
fwiw changing from json to binary doesn't change the underlying physical type, but loses a layer of verification on the type system level. maybe we need to ensure the input data is valid jsonb or be able to handle invalid data. just mark it. we can do this in the future as json type is in a very early experimental stage